### DISCUSSION TITLE
Apple silicon

### DISCUSSION THREAD

**Discussion by tompalmer** [theophanemayaud/video-simili-duplicate-cleaner#119] [General]
Hello, is it planned to port the software on Apple silicon for performance and battery optimisation ?

**Comment by theophanemayaud** [comment]
Hi, thanks for your interest !! Yes it’s been the biggest part of the last few months when I had time to work on the app  It’s ready but I haven’t finished testing that it still works as expected. Progress is tracked under #106

**Reply by tompalmer** [reply]
merci

**Reply by theophanemayaud** [reply]
So @tompalmer after the last fiasco of 1.10.0 rolled back in 1.10.1 because of multiple crashes, I just released v1.11.0 which again includes native apple silicon binaries and also universal ffmpeg libraries ! Hope it makes it as fast on your machine as it made it on mine, it is a great improvement :D Thanks again for the discussion !
